src/commands/ProtectionsCommands.ts

```diff
@@ -92,10 +91,10 @@ async function _execConfigChangeProtection(mjolnir: Mjolnir, parts: string[], ac
         return `Failed to set setting: ${e.message}`;
     }
 
-    const enabledProtections = Object.fromEntries(mjolnir.enabledProtections.map(p => [p.name, p]));
-    if (protectionName in enabledProtections) {
+    const loadedProtection = mjolnir.protections.get(protectionName)
```

Comment:
@Yoric small additional review request: I've made a small change here, given we now don't need to check if a protection is enabled since it will be in memory regardless

Reply:
As far as I can tell, that works.

Nothing beats tests, as usual :)